next.config.mjs

```diff
@@ -3,6 +3,9 @@ const nextConfig = {
   env: {
     NEXT_DEV_FIREBASE_CONFIG: process.env.NEXT_DEV_FIREBASE_CONFIG,
```

Comment:
Have you tried using `NEXT_PUBLIC`? From a security stand point its always best to have API calls be done via an API route, but since we are using React Fire it lowkey has to be done client side, so why not use `process.env.NEXT_PUBLIC_FIREBASE_CONFIG` from `.env.local`?